### ISSUE TITLE
Getting `Error: NavigatorUserMediaError` when trying `demo.html` on local machine

### ISSUE THREAD

**Issue by pavitran** [andyet/SimpleWebRTC#632]
`Error: NavigatorUserMediaError
    at https://simplewebrtc.com/latest-v2.js:2:8069
    at sendResponseAndClearCallback (extensions::messaging:415:9)
    at disconnectListener (extensions::messaging:433:9)
    at EventImpl.dispatchToListener (extensions::event_bindings:403:22)
    at Event.publicClassPrototype.(anonymous function) [as dispatchToListener] (extensions::utils:140:26)
    at EventImpl.dispatch_ (extensions::event_bindings:387:35)
    at EventImpl.dispatch (extensions::event_bindings:409:17)
    at Event.publicClassPrototype.(anonymous function) [as dispatch] (extensions::utils:140:26)
    at dispatchOnDisconnect (extensions::messaging:376:27)`

**Comment by xdumaine** [comment]
Are you trying to load it from a file url? You need to serve it as http(s).

**Comment by pavitran** [comment]
Yes I'm serving it on remote https  server, on firebase hosting to be exact.
this is the link 👉 https://sandbox-71899.firebaseapp.com/demo.html
Everything is working fine except for the screen sharing feature

**Comment by fippo** [comment]
screensharing requires an extension in Chrome. See https://github.com/otalk/getscreenmedia

**Comment by pavitran** [comment]
@fippo I've already have the extension installed with `content_scripts` `matches` set to my website url.
